bin/insertCategories.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO "source_category"
+(title, enabled)
+VALUES
+('Languages', true),
+('Web', true),
+('Mobile', true),
```

Comment:
You will run this once to insert initial categories?

Reply:
Yes. Just to have a reference on what we are trying to run :shipit: